config/webpack.config.prod.js

```diff
@@ -115,7 +115,7 @@ module.exports = {
       // use the "style" loader inside the async code so CSS from them won't be
       // in the main CSS file.
       {
-        test: /\.css$/,
+        test: /\.scss$/,
```

Comment:
ok, I'd edited that to `/\.s?css/` on my local but it didn't make it into the commit, my bad. I agree with your change.